README update, json dataset query path

DanielReid opened the pull request.

DanielReid committed:
small change

DanielReid committed:
Merge branch 'master' of https://github.com/gertvv/jena-es

Conflicts:
	.gitignore
	DESIGN.md

DanielReid committed:
Merge branch 'master' of https://github.com/gertvv/jena-es

DanielReid committed:
Merge branch 'master' of https://github.com/gertvv/jena-es

DanielReid committed:
Merge branch 'master' of github.com:connorstroomberg/jena-es

DanielReid committed:
update readme package instructions

gertvv commented:
I'm confused... What's with the .iml file, and why do the instructions need updating?

DanielReid committed:
remove jetbrains file, ignore in future

DanielReid commented:
My bad, dunno how that .iml got in there. Cleaned it out.

to build the jar with all dependencies correctly it's necessary to use `mvn install spring-boot:repackage` so I updated the readme.

There's also an extra /datasets route in there from Connor's commits.

gertvv commented:
Not sure about the "install" part but I guess I can believe the `spring-boot:repackage`.
